src/LevelNav.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import './LevelNav.css';
+
+import { Link } from "react-router-dom"
+import {BiChevronLeft, BiChevronRight} from "react-icons/bi"
+
+class LevelNav extends React.Component {
+  render(){
+    let navClass = "LevelNav"
+
+    if(this.props.show) {
+       navClass = 'LevelNav open'
+    }
+
+    let levelNum = this.props.levelNum;
+    if (levelNum === undefined){
+      levelNum = 1;
+    }
+
+    let maxLevel = 4;
```

Comment:
Since this is something we're probably gonna be changing pretty often, it would be best to put this in a constants.js, which you can put in a folder called constants in src.